templates/todo/projects/nodejs-mongo/.repo/bicep/repo.yaml

```diff
@@ -56,8 +56,13 @@ repo:
     - from: ../../../../../common
       to: ./
       ignore:
-      - .devcontainer/**/*
-      - "infra/**/*"
+        - .github/**/*
+        - .devcontainer/**/*
+        - "infra/**/*"
+
+    # Github workflows for bicep
+    - from: ../../../common/.github/workflows/bicep
```

Comment:
should not the path be ../../../../../common/.github/workflows/bicep ? this is causing the file to not copied on the generated template PR branch https://github.com/Azure-Samples/todo-nodejs-mongo/compare/staging...pr/650

Reply:
Fixed.